plugins/reporters/web-app-template/src/components/IssuesTable.jsx

```diff
@@ -45,7 +45,11 @@ import SeverityTag from './SeverityTag';
 import { getColumnSearchProps } from './Shared';
 
 // Generates the HTML to display issues as a table
-const IssuesTable = ({ webAppOrtIssues = [], showExcludesColumn = true }) => {
+const IssuesTable = ({
```

Comment:
Commit message nit: Could backtick `severeIssueThreshold` and `severeRuleViolationThreshold`.